src/Mod/Draft/Resources/ui/preferences-draftsnap.ui

```diff
@@ -527,30 +518,21 @@
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="toolTip">
-      <string>Draft edit preferences</string>
+      <string>Draft Edit gui command preferences</string>
```

Comment:
Should gui here be explicit "Graphical User Interface" or at least capitalized 'GUI' ?

Reply:
In FreeCAD, the phrase "Gui Command" basically describes every button and menu entry which we can click on, because all such tools are based on the C++ class `Gui::Command`. So, in this case, Carlo is using the technical term. However, I think we shouldn't use such technical descriptions in user documentation because it may be confusing.

So, I would reword this to be simpler, like "Draft Edit preferences", or "Preferences for the Draft Edit tool".

Reply:
Ok, I probably agree. Let's go with "Draft Edit preferences"